images/wkdev_sdk/Containerfile

```diff
@@ -80,6 +80,16 @@ RUN sed -i 's/^Types: deb$/Types: deb deb-src/' /etc/apt/sources.list.d/ubuntu.s
     rm -rf WebKit && \
     ${APT_AUTOREMOVE}
 
+# Add Rust environment.
+ENV RUSTUP_HOME="/opt/rust" \
+    CARGO_HOME="/opt/rust" \
+    PATH="/opt/rust/.cargo/bin:${PATH}"
+
+RUN rustup default stable && \
+    rustup component remove rust-docs && \
+    cargo install --root /usr/local --version 0.8.1 --locked sccache && \
+    cargo install --root /usr/local cargo-c
```

Comment:
If the entire toolchain (`rustc`, `cargo`, etc), is in `/opt/rust` then we probably don't need to use `/usr/local` for this?